locales/en/messages.json

```diff
@@ -1456,40 +1462,29 @@
         "description": "One of the modes of anti gravity"
     },
     "pidTuningAntiGravityGain": {
-        "message": "Anti Gravity Gain"
+        "message": "Gain"
     },
     "pidTuningAntiGravityThres": {
-        "message": "Anti Gravity Threshold"
+        "message": "Threshold"
     },
-    "pidTuningDMinHelp": {
-        "message": "D Min provides a way to have a lower level of D in normal flight and a higher level for quick manoeuvres that might cause overshoot, like flips and rolls. It also brings D up during prop wash. The craft will have a dynamic D Term varying from D Min (under normal flight) to the standard D (under fast roll and flips). A value of zero will disable the feature. The Gain determines how strongly D is boosted during quick stick inputs. The Advance speeds up of onset of the boost effect. This can be helpful if you have overshoot with very high rate flips on very responsive quads.",
-        "description": "Help message for D Min feature"
+    "pidTuningAntiGravityHelp": {
+        "message": "Anti Gravity boosts the I term when fast throttle changes are detected. Higher gain values provide stability and better attitude hold when you pump the throttle."
     },
     "pidTuningDMin": {
         "message": "D Min",
        "description": "Table header of the D Min feature in the PIDs tab" 
     },
-    "pidTuningDMinRoll": {
-       "message": "Roll",
-       "description": "Axis to apply D Min feature" 
-    },
-    "pidTuningDMinPitch": {
-       "message": "Pitch",
-       "description": "Axis to apply D Min feature"
-    },
-    "pidTuningDMinYaw": {
-       "message": "Yaw",
-       "description": "Axis to apply D Min feature"
-    },
     "pidTuningDMinGain": {
        "message": "Gain",
        "description": "Gain of the D Min feature"
     },
     "pidTuningDMinAdvance": {
        "message": "Advance",
-       "description": "Advane of the D Min feature"
+       "description": "Advance of the D Min feature"
+    },
+    "pidTuningDMinHelp": {
+        "message": "Gain adjusts how fast D min gets up to its maximum value and is based on gyro to determine sharp moves and propwash events. Advance makes D min go up earlier by using setpoint instead of gyro to determine sharp moves."
```

Comment:
This is a general comment, not attached to any particular message. We are trying to give context to all the messages for the translations. Maybe some of them not need it, but if if in doubt, is a good habit to add a description with the context , the element and or the explanation of the message.

Reply:
I will add descriptions for messages where actual messages lack context.